server/src/test/java/com/objectcomputing/checkins/services/skills/combineskills/CombineSkillsControllerTest.java

```diff
@@ -0,0 +1,316 @@
+                () -> skillClient.toBlocking().exchange(skillRequest2, Map.class));
+
+        assertNotNull(responseException.getResponse());
+        assertEquals(HttpStatus.NOT_FOUND,responseException.getStatus());
+
+    }
+
+    @Test
+    public void testPOSTCombine2SkillsIntoOneCheckMemberSkills() {
+
+        MemberProfile memberProfile1 = createADefaultMemberProfile();
+        MemberProfile memberProfile2 = createAnUnrelatedUser();
+
+        Skill skill1 = createADefaultSkill();
+        Skill skill2 = createASecondarySkill();
+
+        String skillLevel = "Guru";
+        LocalDate lastUsedDate = LocalDate.now();
+
+        MemberSkill memberSkill1 = createMemberSkill(memberProfile1,skill1, skillLevel, lastUsedDate);
+
+        skillLevel = "Guru Plus";
+        lastUsedDate = LocalDate.now();
+        MemberSkill memberSkill2 = createMemberSkill(memberProfile1,skill2, skillLevel, lastUsedDate);
+
+        skillLevel = "Plano Guru";
+        lastUsedDate = LocalDate.now();
+        MemberSkill memberSkill3 = createMemberSkill(memberProfile2,skill2, skillLevel, lastUsedDate);
+
+        UUID[] newSkillsArray = new UUID[2];
+        newSkillsArray[0] = skill1.getId();
+        newSkillsArray[1] = skill2.getId();
+        CombineSkillsDTO combineSkillsDTO =
+                new CombineSkillsDTO("New Skill", "New Skill Desc", newSkillsArray);
+
+        final MutableHttpRequest<CombineSkillsDTO> request = HttpRequest.POST("", combineSkillsDTO).basicAuth(ADMIN_ROLE, ADMIN_ROLE);
+        final HttpResponse<Skill> response = client.toBlocking().exchange(request, Skill.class);
+
+        Skill returnedSkill = response.body();
+
+        assertEquals(HttpStatus.CREATED, response.getStatus());
+        assertEquals(combineSkillsDTO.getName(), response.body().getName());
+        assertEquals(returnedSkill.getName(), response.body().getName());
+        assertEquals(String.format("%s/%s", request.getPath(), returnedSkill.getId()), response.getHeaders().get("location"));
+//
+//        final MutableHttpRequest<Object> skillRequest = HttpRequest.GET(String.format("/%s", skill1.getId())).basicAuth(ADMIN_ROLE, ADMIN_ROLE);
+//        HttpClientResponseException responseException = assertThrows(HttpClientResponseException.class,
+//                () -> skillClient.toBlocking().exchange(skillRequest, Map.class));
+//
+//        assertNotNull(responseException.getResponse());
+//        assertEquals(HttpStatus.NOT_FOUND,responseException.getStatus());
+//
+//        final MutableHttpRequest<Object>  skillRequest2 = HttpRequest.GET(String.format("/%s", skill2.getId())).basicAuth(ADMIN_ROLE, ADMIN_ROLE);
+//        responseException = assertThrows(HttpClientResponseException.class,
+//                () -> skillClient.toBlocking().exchange(skillRequest2, Map.class));
+//
+//        assertNotNull(responseException.getResponse());
+//        assertEquals(HttpStatus.NOT_FOUND,responseException.getStatus());
```

Comment:
Dead code above...bury it before it smells up the place.

Reply:
oops! haven't done that in a while. It has been removed